XamarinCommunityToolkit/RegularExpressions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace XamarinCommunityToolkit
+{
+    public static class RegularExpressions
+    {
+        internal static string Email => @"^(?("")("".+?(?<!\\)""@)|(([0-9a-z]((\.(?!\.))|[-!#\$%&'\*\+/=\?\^`\{\}\|~\w])*)(?<=[0-9a-z])@))(?(\[)(\[(\d{1,3}\.){3}\d{1,3}\])|(([0-9a-z][-0-9a-z]*[0-9a-z]*\.)+[a-z0-9][\-a-z0-9]{0,22}[a-z0-9]))$";
+        internal static string Password => @"^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$";
```

Comment:
This one is a tricky one for me. I'm not sure if we should be providing password validation rules as that is an entirely different world on its own and people in infosec might have strong opinions on it. I would perhaps take that specific one out.